flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/KafkaSource.java

```diff
@@ -108,6 +108,15 @@
         return new KafkaSourceBuilder<>();
     }
 
+    /**
+     * Returns the props for the Kafka Source.
+     *
+     * @return props for the Kafka Source.
+     */
+    public Properties getProps() {
```

Comment:
Should this method be added to `KafkaSourceBuilder` instead of in `KafkaSource`? And this method probably should be package private.

Reply:
I think  getProps() and getBoundedness() in KafkaSource are similar,we need get props from KafkaSource object.